src/overlays/actors/ovl_En_Skb/z_en_skb.c

```diff
@@ -64,57 +72,496 @@ static ColliderJntSphInit D_80AFE068 = {
+                    }
+                } else {
+                    phi_v1 = 8;
+                    if (this->actor.colChkInfo.damageEffect == 7) {
+                        scale = this->actor.scale.y * 7500.0f;
+                        for (i = 4; i >= 0; i--) {
+                            flamePos = this->actor.world.pos;
+                            flamePos.x += Rand_CenteredFloat(20.0f);
+                            flamePos.z += Rand_CenteredFloat(20.0f);
+                            flamePos.y += (Rand_ZeroOne() * 25.0f);
+                            EffectSsEnFire_SpawnVec3f(globalCtx, &this->actor, &flamePos, scale, 0, 0, -1);
+                        }
+                        phi_v1 = 25;
+                    }
+                    func_8003426C(&this->actor, 0x4000, 0xFF, 0, phi_v1);
+                    if (!Actor_ApplyDamage(&this->actor)) {
+                        func_80AFD7B4(this, globalCtx);
+                        return;
+                    }
+                    player = PLAYER;
+                    if (this->unk_283 == 0) {
+                        if ((this->actor.colChkInfo.damageEffect == 0xD) ||
+                            ((this->actor.colChkInfo.damageEffect == 0xE) &&
+                             ((player->swordAnimation >= 4 && player->swordAnimation <= 11) ||
+                              (player->swordAnimation == 20 || player->swordAnimation == 21)))) {
+                            func_80032E24(&this->unk_28C, 2, globalCtx);
+                            this->unk_283 = 1;
+                        }
+                    }
+                    func_80AFD644(this);
+                }
+            }
+        }
+    }
+}
+
+void EnSkb_Update(Actor* thisx, GlobalContext* globalCtx) {
+    EnSkb* this = THIS;
+    s32 pad;
+
+    func_80AFD968(this, globalCtx);
+    Actor_MoveForward(&this->actor);
+    Actor_UpdateBgCheckInfo(globalCtx, &this->actor, 15.0f, 30.0f, 60.0f, 0x1D);
+    this->actionFunc(this, globalCtx);
+    this->actor.focus.pos = this->actor.world.pos;
+    this->actor.focus.pos.y += (3000.0f * this->actor.scale.y);
+    if (this->unk_281 != 0) {
+        CollisionCheck_SetAT(globalCtx, &globalCtx->colChkCtx, &this->collider.base);
+    }
+
+    if (this->unk_280 >= 3) {
+        if ((this->actor.colorFilterTimer == 0) || ((this->actor.colorFilterParams & 0x4000) == 0)) {
+
+            CollisionCheck_SetAC(globalCtx, &globalCtx->colChkCtx, &this->collider.base);
+        }
+    }
+    CollisionCheck_SetOC(globalCtx, &globalCtx->colChkCtx, &this->collider.base);
+}
+
+s32 EnSkb_OverrideLimbDraw(GlobalContext* globalCtx, s32 limbIndex, Gfx** dList, Vec3f* pos, Vec3s* rot, void* thisx) {
+    EnSkb* this = THIS;
+    s16 color;
+    s16 pad[2];
+
+    if (limbIndex == 11) {
+        if ((this->unk_283 & 2) == 0) {
+            OPEN_DISPS(globalCtx->state.gfxCtx, "../z_en_skb.c", 972);
+            color = ABS((s16)(Math_SinS((globalCtx->gameplayFrames * 0x1770)) * 95.0f)) + 0xA0;
```

Comment:
we use decimal for colors so I would have `160` here
```suggestion
            color = ABS((s16)(Math_SinS((globalCtx->gameplayFrames * 0x1770)) * 95.0f)) + 160;
```